apps/src/code-studio/appRedux.js

```diff
@@ -0,0 +1,31 @@
+const SET_APP_LOAD_STARTED = 'SET_APP_LOAD_STARTED';
+const SET_APP_LOADED = 'SET_APP_LOADED';
```

Comment:
Similar to @madelynkasula's other comment, I think you'll want to prefix these with the reducer name (`app/SET_APP_LOAD_STARTED`)

Reply:
Thanks Sanchit! Just updated.